### PR TITLE
Create docker-compose-pgsql-ssession

### PR DESCRIPTION
**Pull request by shadinua**
Now we can launch the whole application stack with one command:
```
docker-compose -f docker/docker-compose-pgsql-ssession.yml up
```
This will create `docker/.acrakeys` directory structure, generate all key pairs,
put them to appropriate servcies' directories and launch all components.
See details in `README.md`.

* Created and documented docker-compose-pgsql-ssession.yml
* Optimized acra-build.dockerfile: remove unnecessary git clone
* Refined arguments in docker images
* Removed obsolete `docker-compose.yml`
* Added `README.md`

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+## Quick launch
+
+!!!!!!!!!!!!!!!!!!!!!!!!!!!
+!!! INSECURE, TEST ONLY !!!
+!!!!!!!!!!!!!!!!!!!!!!!!!!!
+```
+docker-compose -f docker/<compose_file_name>.yml up
+```
+This will create `docker/.acrakeys` directory structure, generate all key pairs,
+put them to appropriate servcies' directories and launch all components.
```
Comment (Lagovas) [inline docker/README.md]: `servcies` -> `services`

File: docker/README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+Now you can connect to 9494/tcp (acraproxy) or 5432/tcp (postgresql).
+
+
+## Normal launch
+
+Please specify ACRA_MASTER_KEY:
+```
+export ACRA_MASTER_KEY=$(echo -n "My_Very_Long_Key_Phrase_ge_32_chars" | base64)
```
Comment (Lagovas) [inline docker/README.md]: if `ge` == great equal, then we need to update our master key check and change `==` sign to `>=`. I think I will do it in a future

File: docker/docker-compose-pgsql-ssession.yml [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+            - acra_genkeys_proxy
+        network_mode: "none"
+        environment:
+            ACRA_MASTER_KEY: ${ACRA_MASTER_KEY:-UHZ3VUNNeTJ0SEFhbWVjNkt4eDdVYkc2WnNpUTlYa0E=}
+        volumes:
+            - ./.acrakeys:/keys
+        command: >-
+            --client_id=${ACRA_CLIENT_ID:-testclientid}
+            --storage
+            --output=/keys/acrawriter
```
Comment (Lagovas) [inline docker/docker-compose-pgsql-ssession.yml]: acrawriter uses public key that related to private key stored on acraserver side
so will be correct to set:
```
--output=/keys/acraserver
--output_public=/keys/acrawriter
```
Acraproxy doesn't need to has access to acrawriter's public key

File: docker/README.md (hunk 1 above)
Comment (shadinua) [reply docker/README.md]: Thank you, fixed.

File: docker/README.md (hunk 2 above)
Comment (shadinua) [reply docker/README.md]: OK, so I keep this wording.

File: docker/docker-compose-pgsql-ssession.yml (hunk 3 above)
Comment (shadinua) [reply docker/docker-compose-pgsql-ssession.yml]: Thank you for explain, fixed.

**Pull request merged**